mercury.spec

```diff
@@ -21,6 +21,7 @@ URL:		http://mercury-hpc.github.io/documentation/
 Source0:	https://github.com/mercury-hpc/%{name}/archive/v%{version}.tar.gz
 Patch1:		https://github.com/mercury-hpc/mercury/compare/v2.0.0a1..%{mercury_commit}.patch
 Patch2:		https://github.com/mercury-hpc/mercury/compare/c2c262813811c3ede28ee32fdebbffd417a7cb80..6cbd9d4402db6a34dfb7e9dbf2e826ee90c6a64a.patch
```

Comment:
Alex just uploaded a patch to remove this patch from the build...should we do that here too?

Reply:
I will rollback this patch and create a new one